localization/english/load_tips/load_tips_l_english.yml

```diff
@@ -62,8 +62,8 @@
  LOADING_TIP_60:0 "#T Did you know?#!\n\nThe Monkey King is a hozen who was a companion of Shaohao, the last emperor of Pandaria."
  LOADING_TIP_61:0 "#T Did you know?#!\n\nCho'gall was once claimed to be one of the first ogre-magi, or indeed the first."
  LOADING_TIP_62:0 "“I have watched the other races... I have seen their squabbling, their ruthlessness. Their wars do nothing but scar the land, and drive the wild things to extinction. No, they cannot be trusted. Only beasts are above deceit.”\n #bold —Rexxar#!"
- LOADING_TIP_63:0 "#T Did you know?#!\n\nRexxar claims to be, in his own words, the "last son of the Mok'Nathal""
- LOADING_TIP_64:0 "#T Did you know?#!\n\nThe Nathrezim, also known as the Dread Lords, are most commonly associated with the Burning Legion. However, there are a few who serve the Army of the Light."
+ LOADING_TIP_63:0 "#T Did you know?#!\n\nRexxar claims to be, in his own words, the “last son of the Mok'Nathal”"
+ LOADING_TIP_64:0 "#T Did you know?#!\n\n There is rumored to be a gateway that appears every thirteenth full moon that bridges the gap between Azeroth and the underworld, and it is during these brief moments that the daemons can come to this place."
```

Comment:
Unneeded space after line break

Reply:
You have not removed the space after the line break here. Fix it.

Reply:
I did